markdown/usage/nf_core_tutorial.md

```diff
@@ -288,63 +397,132 @@ nf-core lint /path/to/pipeline
+Fork the repository to your user and open a PR to your new branch with a really (really!) tiny dataset.
+Once merged, set up the `conf/test.config` file in your pipeline to refer to the URLs for your test data.
+
+These test datasets are used by the automated continuous integration tests.
+The systems that run these tests are _extremely_ limited in the resources that they have available.
+Typically, the pipeline should be able to complete in around 10 minutes and use no more than 6-7 GB memory.
+To achieve this, input files and reference genomes need to be very tiny.
+If possible, a good approach can be to use PhiX or Yeast as a reference genome.
+Alternatively, a single small chromosome (or part of a chromosome) can be used.
+If you are struggling to get the tests to run, ask for help on Slack.
+
+When writing `conf/test.config` remember to define all required parameters so that the pipeline will run with only `-profile test`.
+Note that remote URLs cannot be traversed like a regular file system - so glob file expansions such as `*.fa` will not work.
+
+### GitHub Actions configuration
 
-These test datasets are used by the automated continuous integration tests. The systems that run these tests are _extremely_ limited in the resources that they have available. Typically, the pipeline should be able to complete in around 10 minutes and use no more than 6-7 GB memory. To achieve this, input files and reference genomes need to be very tiny. If possible, a good approach can be to use PhiX or Yeast as a reference genome. Alternatively, a single small chromosome (or part of a chromosome) can be used. If you are struggling to get the tests to run, ask for help on Slack.
+The automated tests with GitHub Actions are configured in the `.github/worfklows` folder that is generated by the template.
+Each file Each file (`branch.yml`, `ci.yml` and `linting.yml`) defines several tests: branch protection, running the pipeline with the test data, linting the code with `nf-core lint`, linting the syntax of all Markdown documentation and linting the syntax of all YAML files.
 
-When writing `conf/test.config` remember to define all required parameters so that the pipeline will run with only `-profile test`. Note that remote URLs cannot be traversed like a regular file system - so glob file expansions such as `*.fa` will not work.
+#### `branch.yml`
 
-### Travis CI configuration
-The automated tests with Travis CI are configured in the `.travis.yml` file that is generated by the template. The `script` block defines three tests: linting the code with `nf-core lint`, linting the syntax of all Markdown documentation and running the pipeline with the test data.
+This is already set up and should not be touched.
+It will check that a PR is made on `dev` unless it's for a release or a patch.
```

Comment:
```suggestion
It will check that pull-requests going to the nf-core repo `master` branch only come from the `dev` or `patch` branches (for releases).
```